Acrolinx.Sidebar/AcrolinxSidebar.cs

```diff
@@ -513,211 +490,197 @@ internal bool FireOpenBrowser(string url)
-                        webBrowser.Focus();
-                        frame.Document.ActiveElement.Focus();
-                        break;
-                    }
+                    webView2.CoreWebView2.Settings.AreBrowserAcceleratorKeysEnabled = true;
+                    webView2.CoreWebView2.Settings.AreDefaultContextMenusEnabled = true;
                 }
             }
-            FixFocusTimer.Enabled = false;
-        }
-
-        private void ShowHideServerSelectorIfServerAddressParameterSet(string serverAddress)
-        {
-            if (!string.IsNullOrWhiteSpace(serverAddress))
-            {
-                this.ServerAddress = serverAddress;
-                this.ShowServerSelector = false;
-            }
-            else if (string.IsNullOrWhiteSpace(this.ServerAddress))
-            {
-                this.ShowServerSelector = true;
+            catch (Exception e) {
+                Logger.AcroLog.Error("Unable to parser EnableContextMenu to integer." + e.Message);
             }
         }
 
-        private void SetUiError()
+        private void CoreWebView2_WebResourceRequested(object sender, CoreWebView2WebResourceRequestedEventArgs e)
         {
-            labelImage.Text = "\n\n\nOops, something went wrong with loading the Sidebar. Check the log file for any errors.";
-            labelImage.TextAlign = ContentAlignment.TopLeft;
+            Logger.AcroLog.Info("Sidebar requested from: " + e.Request.Uri.ToString());
         }
 
-        private void AcrolinxSidebar_Resize(object sender, EventArgs e)
+        private void webView2_Resize(object sender, EventArgs e)
         {
             AdjustSidebarZoomLevelByWidth();
         }
 
         private void AdjustSidebarZoomLevelByWidth()
         {
-            try
+            if (webView2.Parent == null)
             {
-                if (webBrowser.ReadyState != WebBrowserReadyState.Complete)
-                {
-                    return;
-                }
-
-                object ax = webBrowser.ActiveXInstance;
-                if (ax == null)
-                {
-                    return;
-                }
-
-                dynamic activeX = ax;
-                const int OLECMDID_OPTICAL_ZOOM = 63;
-                const int OLECMDEXECOPT_DONTPROMPTUSER = 2;
-                int zoomFactor = (int)(this.Width * 100 * Util.GraphicUtil.GetScaling() / 300);
-
-                activeX.ExecWB(OLECMDID_OPTICAL_ZOOM, OLECMDEXECOPT_DONTPROMPTUSER, zoomFactor, IntPtr.Zero);
+                return;
             }
-            catch (Exception e)
+
+            var scaling = Util.GraphicUtil.GetScaling();
+            var sidebarConstantWidth = 300;
+            var width = webView2.Parent.ClientSize.Width;
+            if (width > 0)
             {
-                Logger.AcroLog.Warn(e.Message);
+                webView2.ZoomFactor = webView2.Parent.ClientSize.Width / (sidebarConstantWidth * scaling);
             }
         }
+
+        public void Quit()

```

Comment:
Rename it to stop

Reply:
Also add comment to explain when integrator can use it and highlight its an optional call.

Reply:
Or remove this function if its not adding any handling for integrator.

Reply:
removed the quit function